pandas/core/arrays/base.py

```diff
@@ -44,10 +44,12 @@ class ExtensionArray(object):
     * copy
     * _concat_same_type
 
-    An additional method is available to satisfy pandas' internal,
-    private block API.
+    A default repr displaying the type, (truncated) data, length,
+    and dtype is provided. It can be customized or replaced by
+    by overriding:
 
-    * _formatting_values
+    * _formatter
+    * __repr__
```

Comment:
Maybe also mention that `_formatter` is used to determine how the values are shown in Series / DataFrame as well? (I would even say that it is its main purpose)